--- host.py	2018-09-11 15:39:28.874253398 +0000
+++ /home/modep/host.new	2020-06-11 22:36:34.571706506 +0000
@@ -1439,6 +1439,20 @@
         pluginData['ports'][symbol] = value
         self.send_modified("param_set %d %s %f" % (instance_id, symbol, value), callback, datatype='boolean')
 
+    def pi_stomp_param_get(self, port):
+        instance, symbol = port.rsplit("/", 1)
+        instance_id = self.mapper.get_id_without_creating(instance)
+        pluginData  = self.plugins[instance_id]
+
+        if symbol == ":bypass":
+            return pluginData['bypassed']
+
+        if symbol in pluginData['designations']:
+            print("ERROR: Trying to modify a specially designated port '%s', stop!" % symbol)
+            return
+
+        return pluginData['ports'][symbol]
+
     def set_position(self, instance, x, y):
         instance_id = self.mapper.get_id_without_creating(instance)
         pluginData  = self.plugins[instance_id]
