--- webserver.py.orig	2020-06-15 11:31:53.000000000 +0100
+++ webserver.py	2020-08-14 21:56:54.429424077 +0100
@@ -938,6 +938,25 @@
 
         self.write(ok)
 
+class EffectParameterSetPiStomp(JsonRequestHandler):
+    @web.asynchronous
+    @gen.engine
+
+    def post(self, port):
+        data = json.loads(self.request.body.decode("utf-8", errors="ignore"))
+        value   = float(data['value'])
+
+        ok = yield gen.Task(SESSION.pi_stomp_parameter_set, port, value)
+        self.write(ok)
+
+class EffectParameterGetPiStomp(JsonRequestHandler):
+    @web.asynchronous
+    @gen.engine
+
+    def get(self, port):
+        value = SESSION.host.pi_stomp_param_get(port)
+        self.write(value)
+
 class EffectPresetLoad(JsonRequestHandler):
     @web.asynchronous
     @gen.engine
@@ -2101,6 +2120,9 @@
         elif filetype == "sfz":
             return ("SFZ Instruments", (".sfz",))
 
+        elif filetype == "tapf":
+            return ("Amplifier Profiles", (".tapf",))
+
         else:
             return (None, ())
             
@@ -2167,6 +2189,8 @@
             # plugin parameters
             (r"/effect/parameter/address/*(/[A-Za-z0-9_:/]+[^/])/?", EffectParameterAddress),
             (r"/effect/parameter/set/?", EffectParameterSet),
+            (r"/effect/parameter/pi_stomp_set/*(/[A-Za-z0-9_:/]+[^/])/?", EffectParameterSetPiStomp),
+            (r"/effect/parameter/pi_stomp_get/*(/[A-Za-z0-9_:/]+[^/])/?", EffectParameterGetPiStomp),
 
             # plugin presets
             (r"/effect/preset/load/*(/[A-Za-z0-9_/]+[^/])/?", EffectPresetLoad),